src/app/Table.elm

```diff
@@ -1,29 +1,95 @@
-module Table exposing (view, init, Msg)
+module Table exposing (view, init, initWithPlayersAndCup)
+
+{-|
+
+   This module handles the table of players,
+   and everything relating to that.
+
+-}
 
 import Html exposing (..)
+import Html.Events exposing (..)
+import Types exposing (PlayerID)
+import TableMsg exposing (Msg(..))
+import Dice
 
 
-{-
+type alias AppMsg =
+    Types.Msg
 
-   This module handles the view rendering for the table of players
-   and everything relating to that.
 
--}
+type alias PlayerName =
+    String
+
+
+type alias DiceCount =
+    Int
+
+
+type alias TablePlayer =
```

Comment:
This might seem like repetition, but I think it's good to keep the server types different to the front end types.

For example, we might decide to add a random colour to a player when they're created in our model, to differentiate the players in the browser for the user, but the server wouldn't necessarily have such information.

Or, we might put rendering positions in here. Either way, a `TablePlayer` is potentially quite different to the `Player`, even though it's not *yet*. :)